lib/output.js

```diff
@@ -1185,6 +1200,20 @@ function OutputStream(options) {
         output.print(self.getValue());
     });
     DEFPRINT(AST_String, function(self, output){
+        // Check if there is a need to break directive by inserting an additional semicolon
+        if (in_directive) {
+            var str = output.encode_string(self.getValue(), self.quote);
+            // Only output semicolons if string might be a valid directive
+            if (str.indexOf("\\") === -1) {
+                var code = output.get();
+                // Extra semicolon might be needed to break directive prologue
+                if (!NEED_SINGLE_SEMICOLON.test(code)) {
+                    output.force_semicolon();
+                }
+                output.force_semicolon();
```

Comment:
In case a newline is used, so the automatic semicolon insertion would work. Then a single semicolon won't be enough to break the directive prologue rule.